packages/common-all/src/types/configs/dev/DendronDevConfig.ts

```diff
@@ -26,6 +26,12 @@ export type DendronDevConfig = {
    * Enable new preview as default
    */
   enablePreviewV2?: boolean;
+  /** Force the use of a specific type of watcher.
+   *
+   * - plugin: Uses VSCode's builtin watcher
+   * - engine: Uses the engine watcher, watching the files directly without VSCode
+   */
+  forceWatcherType?: "plugin" | "engine";
```

Comment:
@hikchoi - we had duplicated types for `DendronDevConfig` - the only difference between them is that the one we were exporting had this `forceWatcherType` property.  Is it fine to include it / remove it? 

FWIW - longer term I would prefer to remove it, but it looks like there are a few references to it in plugin-core and elsewhere.

Reply:
this seems to be added to the old config type while config migration was in process (and the dev namespace itself was never migrated because there was nothing to migrate).

so `DendronDevConfig.ts` was never exported and was there as a placeholder during config migration. (plan was to clean up the old type that was actually exported and export this instead when that happens)

so yeah, for now with this change we should do this.

tangentially, I think this was added as part of Native workspaces, but after a quick skim through the code I don't think we need chokidar to watch files at all, so we can remove this in the future.

Sorry about the confusion with these remnant config types. Looking at the stats I think we can proceed with closing the loop on config migration on October finally so I'll be cleaning the rest of this up.